src/Features/CSharp/Portable/DuplicateMethod/CSharpDuplicateMethodCodeRefactoringProvider.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Composition;
+using Microsoft.CodeAnalysis.CodeRefactorings;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.DuplicateMethod;
+
+namespace Microsoft.CodeAnalysis.CSharp.DuplicateMethod
```

Comment:
The main use case is duplicating test methods. I'm not sure if it makes sense to offer this for all constructs. /cc @sharwell since he suggested to verify the opposite.